workflows/ChIP-seq/internals.snakefile

```diff
@@ -26,10 +26,10 @@ def get_control_name(sample):
     Return False if given ChIP-seq sample has no control
     """
     if sample in chip_samples_w_ctrl:
-        if 'control_name' in chip_dict[sample] and chip_dict[sample]['control_name'] != None:
-            return(chip_dict[sample]['control_name'])
+        if 'control' in chip_dict[sample] and chip_dict[sample]['control'] != None:
+            return chip_dict[sample]['control']
         else:
-            return('Input')
+            return False
     else:
```

Comment:
@vivekbhr Can you have a look at this, specifically `return 'Input'`? I presume you really meant `return False`. Otherwise this PR is good to go.

Reply:
Previously all control samples were called `Input`. We wanted to return the name `input` only when control sample name is not in the `control_name` dict. Although I think this whole else statement can go away now since if the control sample name is not there in the dict the whole expression should return False.